docs/source/markdown/podman-compose.1.md.in

```diff
@@ -13,6 +13,8 @@ The default compose providers are `docker-compose` and `podman-compose`.  If ins
 
 If you want to change the default behavior or have a custom installation path for your provider of choice, please change the `compose_provider` field in `containers.conf(5)`.  You may also set the `PODMAN_COMPOSE_PROVIDER` environment variable.
 
+By default, `podman-compose` will emit a warning when using an external compose provider (such as `docker-compose`). This warning can be disabled by setting `compose_warning_logs` to false in `containers.conf(5)` or setting the `PODMAN_COMPOSE_WARNING_LOGS` environment variable to false. See the man page for `containers.conf(5)` for more information.
```

Comment:
```suggestion
By default, `podman compose` will emit a warning saying that it executes an external command. This warning can be disabled by setting `compose_warning_logs` to false in `containers.conf(5)` or setting the `PODMAN_COMPOSE_WARNING_LOGS` environment variable to false. See the man page for `containers.conf(5)` for more information.
```

The external provider part is already explained in the two paragraphs above as such I do not think you have to mention it at all. Also it always uses a external provider so saying "when using" is incorrect as this warning always will be there.

Reply:
Updated as suggested